.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: nf-core CI
+# This workflow runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
+on:
+  push:
+    branches:
+      - dev
+  pull_request:
+  release:
+    types: [published]
+
+jobs:
+  test:
```

Comment:
If your pipeline has multiple modes of running, you can also add a few more tests with different parameter settings or modes  to this profile -> see for example the diaproteomics pipeline ci.yml

Reply:
Agree but for now, it's enough with one mode. The full test will do the rest for more advanced options. Thanks